test/test_config/check_policy.yml

```diff
@@ -0,0 +1,38 @@
+# create roles
+- !user permitted_user
+- !host permitted_host
+- !layer permitted_layer
+
+- !user non_permitted_user
+- !host non_permitted_host
+- !layer non_permitted_layer
+
+- !group permitted_group
+- !group non_permitted_group
+
+# add permitted user, host, and layer to permitted_group
+- !grant
+  role: !group permitted_group
+  member:
+  - !user permitted_user
+  - !host permitted_host
+  - !layer permitted_layer
+
+# add non permitted user, host, and layer to non permitted_group
+- !grant
+  role: !group non_permitted_group
+  member:
+  - !user non_permitted_user
+  - !host non_permitted_host
+  - !layer non_permitted_layer
+
+# add resources
+- !policy somepolicy
+- !variable somevariable
+- !webservice somewebservice
```

Comment:
By not adding `somepolicy` and `somewebservice` to a permit statement, only the admin user (or the role loading the policy) has any privileges on these resources. Just making sure that is intentional.

Reply:
yup that was intentional! permitted roles can only access permitted variables. i performed several variations of this test in the test_integration file